MaterialLib/FractureModels/FractureModelBase.h

```diff
@@ -66,6 +66,7 @@ class FractureModelBase
      *
      * @param t           current time
      * @param x           current position in space
+     * @param aperture0   initial fracture's aperture
```

Comment:
I would name this variable as aperture_bm0 instead which physically means the initial mechanical aperture at a certain node or initial mean mechanical aperture at a representative element, thereby leaving some naming space for other relevant variables, for instance, aperture_bh0 standing for initial hydraulic aperture.

Reply:
@endJunction @nagelt The variable "aperture0" introduced here keeps consistent with the pre-existing definition in the header file "FractureProperty.h". Therefore, just leave my previous immature thinking out.